pages/linux/mac2unix.md

```diff
@@ -6,8 +6,8 @@
 
 - Change the line endings of a file:
 
-`mac2unix {{filename}}`
+`mac2unix {{path/to/file}}`
 
 - Create a copy with Unix-style line endings:
 
-`mac2unix -n {{filename}} {{new_filename}}`
+`mac2unix -n {{path/to/file}} {{path/to/file}}`
```

Comment:
To quote myself:
<img width=50 src=https://user-images.githubusercontent.com/35269695/215514195-38218ad8-d859-4025-a7eb-6e3001bf0c9c.jpeg alt="me making a confused/whatever face">